### ISSUE TITLE
Allow AutoModule to have no bias

### ISSUE THREAD

**Issue by yangky11** [twitter/torch-autograd#117]
Hi all,

I'm wrapping my computation into a AutoModule and I do not need bias. 
And I have to do something like `loss = loss + 0 * torch.sum(bias)` in my forward computation. Otherwise there will be an error when calling `module:backward`, since it assumes bias is available.
